tests/by-util/test_cksum.rs

```diff
@@ -1344,3 +1344,40 @@ fn test_check_comment_leading_space() {
         .stdout_contains("foo: OK")
         .stderr_contains("WARNING: 1 line is improperly formatted");
 }
+
+#[cfg(not(windows))]
```

Comment:
why not windows?

Reply:
https://github.com/uutils/coreutils/actions/runs/11424022765/job/31783927716#step:7:3602
seems that windows has a different logic and is guarded by another test
https://github.com/uutils/coreutils/blob/9a6f5521b9d0477192501c406c3f774fe0a0276f/tests/by-util/test_cksum.rs#L1195-L1198

Reply:
could you please document this into the test ?

Reply:
Done.